modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ItemConfiguration.js

```diff
@@ -191,7 +212,12 @@ function ItemConfigurationContent({
 								key={panel.panelId}
 							>
 								<span
-									className="c-inner page-editor__page-structure__item-configuration-tab text-truncate"
+									className={classNames(
+										'c-inner page-editor__page-structure__item-configuration-tab',
+										{
+											'text-truncate': !isExpandedText,
```

Comment:
@ilzamcmed we can't use `text-truncate` due to accessibility recommendation by @marcoscv-work. The full text should always display.

Reply:
Oh, so it's a simpler change than I was expecting.
Thanks @pat270!